server/src/main/java/com/sauron/service/UserServiceImpl.java

```diff
@@ -27,6 +27,12 @@ public UserView login(LoginData loginData) {
                 .orElse(null);
     }
 
+    @Override
+    public UserView get(Long userId) {
+        User user = userRepository.findById(userId).orElseThrow(RuntimeException::new);
+        return convertToUserView(user);
```

Comment:
You can make this syntax a little bit more concise using Optional's map() method

Reply:
Done